workers/data_refinery_workers/processors/salmon.py

```diff
@@ -327,6 +330,7 @@ def _run_salmon(job_context: Dict, skip_processed=SKIP_PROCESSED) -> Dict:
         with transaction.atomic():
             ComputationalResult.objects.select_for_update()
             result.save()
+            SampleResultAssociation.objects.create(sample=job_context['sample'], result=result)
```

Comment:
We are trying to use `SampleResultAssociation.objects.get_or_create` as these are unique now and this will fail if the object already exists

Reply:
Sure. Good point!